lbuild/main.py

```diff
@@ -543,8 +543,7 @@ def main():
 
     except lbuild.exception.LbuildException as error:
         sys.stderr.write('\nERROR: {}\n'.format(error))
-        if args.verbose >= 2:
-            traceback.print_exc()
+        traceback.print_exc()
```

Comment:
```suggestion
        if isinstance(error, lbuild.exception.LbuildException):
            traceback.print_exc()
```

Maybe like this? Otherwise you'll get a stack trace on *all* exceptions including those that are handled and formatted specially, and for those it will be confusing to get a traceback, since the exception is raised somewhere completely unrelated.

Reply:
Why should this except statement catch an exceptions that is handled before?

Reply:
```python
except lbuild.exception.LbuildException as error:
  if isinstance(error, lbuild.exception.LbuildException):
    # ...
```
Sure?

Reply:
Let me try some stuff and I'll come back to this.